registration.html

```diff
@@ -125,6 +125,23 @@ <h3 class="slim-title" for="en-gender">What is your gender?</h3>
 									<input class="slInp hidden" type="text" placeholder="Type your answer here..." id="en-gender">
 								</div>
 							</div>
+							<div class="qGrp">
+								<h3 class="slim-title" for="en-race">What is your race/ethnicity?test</h3>
+								<div class="selectSplit">
+									<select class="slSel" id="en-raceSelect">
+										<option value="select" selected disabled>Select</option>
```

Comment:
Since this is optional, they can submit the form while this is still selected. It'd make more sense to have the value to be `""`